README.md

```diff
@@ -70,3 +54,6 @@ curl -G http://localhost:8086/query?pretty=true \
 * Follow the [getting started guide](http://influxdb.com/docs/v0.9/introduction/getting_started.html) to find out how to install InfluxDB, start writing more data, and issue more queries - in just a few minutes.
 * See the  [HTTP API documentation to start writing a library for your favorite language](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).
 
+## Upgrading
```

Comment:
Do we need to include this information here? I understood this README was about getting someone spinning up a server quickly. Why tell them about upgrading here? It is important, just not clear if this is the best place.